sustaingym/envs/building/stochastic_uncontrollable_generator.py

```diff
@@ -0,0 +1,222 @@
+from __future__ import annotations
+
+from collections.abc import Iterable
+
+import numpy as np
+import pandas as pd
+import scipy
+
+
+class StochasticUncontrollableGenerator():
+    def __init__(self, num_episodes: int | None = None):
+        """
+        Initializes a generator class for the uncontrollable features in BuildingEnv.
+
+        Args:
+            num_episodes: The number of episodes in the provided observation data. If
+                given, instance will treat each row in the observation data as a separate
+                raw dataset to split into seasons and fit distributions to. If None,
+                will use the provided raw data as a single episode.
+        """
```

Comment:
Please move the docstring for the `__init__` function of a class into the docstring of the class itself. That is:

```python
class A:
    """PUT DOCSTRING HERE.

    Args:
        input: ...
    """"

    def __init__(self, ...):
        # code here, without docstring
````